### PR TITLE
ICMSLST-580 Fix XI EORI PDF logic.

### PR DESCRIPTION
**Pull request by MattHolmes123**
Updated logic as follows:
  - FA-DFL shows the XI EORI when the postcode is from NI and the CoC is an EU Member.
  - FA-OIL shows the XI EORI when the postcode is from NI.
  - FA-SIL shows the XI EORI when the postcode is from NI and the CoC is an EU Member.
  - Wood applications should not show XI EORI.
  - Sanctions applications should not show XI EORI.

Will need `python manage.py loaddata country_groups` running after merging to work.

### CODE REVIEW

**Review by marcuspp — APPROVED** [review]

File: web/domains/country/utils.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+# TODO: ICMSLST-2671 Move to CountryGroup model as part of wider country refactor.
+class CountryGroupName(TypedTextChoices):
+    """Country group name constants (taken from V1 live)"""
+
+    ADHOC_APP_COUNTRIES = "Adhoc application countries"
+    CFS = "Certificate of Free Sale Countries"
+    CFS_COM = "Certificate of Free Sale Country of Manufacture Countries"
+    COM = "Certificate of Manufacture Countries"
+    DEROGATION_FROM_SANCTION_COO = "Derogation from Sanctions COOs"
+    EU = ("EU", "All EU countries")
```
Comment (mikeylockwood) [inline web/domains/country/utils.py:16]: Should this be a tuple?

Comment (mikeylockwood) [reply web/domains/country/utils.py:16]: Yes it's a `TypedTextChoice`, just looked strange amongst the strings

**Review by mikeylockwood — APPROVED** [review]

File: pii-ner-exclude.txt [hunk 2]
```diff
@@ -4431,7 +4431,15 @@ dp.dp_id document_pack_id
 Export Certificates - DRAFT
 Export Certificates - REVOKED
 Export Certificates - ACTIVE
+<<<<<<< HEAD
```
[the remark this answers is not in the document]
Comment (MattHolmes123) [reply pii-ner-exclude.txt:4434]: haha oops!

File: web/domains/country/utils.py (hunk 1 above)
Comment (MattHolmes123) [reply web/domains/country/utils.py:16]: I will tidy it all up in that other Country Story. 
I resisted doing too much in this one

**Pull request merged**